src/main/scala/outwatch/dom/VDomModifier.scala

```diff
@@ -46,15 +56,15 @@ final case class ChildStreamReceiver(childStream: Observable[VNode]) extends Chi
 final case class ChildrenStreamReceiver(childrenStream: Observable[Seq[VNode]]) extends ChildVNode
 
 sealed trait VNode_ extends ChildVNode {
-  // TODO: have apply() only on VTree?
-  def apply(args: VDomModifier*): VNode = ???
+  def apply(args: VDomModifier*): VNode
   // TODO: rename asProxy to asSnabbdom?
   def asProxy: VNodeProxy
 }
 
 //TODO: extends AnyVal
-private[outwatch] case class StringNode(string: String) extends VNode_ {
-  val asProxy: VNodeProxy = VNodeProxy.fromString(string)
+private[outwatch] final case class StringNode(string: String) extends VNode_ {
+  override def apply(args: VDomModifier*): VNode = ???
```

Comment:
This should stay as default implementation in `VNode_` and be handled via the TODO instead.